src/client/app/modules/home/home.controller.js

```diff
@@ -20,44 +20,72 @@
   * This page serves as a jumping off point to editing individual routes and makes
   * adding and removing new routes easy
   */
+  //runs activate on startup
 
   function HomeController($http, routes, user, checklist) {
-    var test;
-    this.formInfo = {};
-    this.formInfo.verbs = [];
-    this.routes = routes.routes;
-    this.verbs = ['GET', 'POST', 'PUT', 'DELETE', 'OPTIONS'];
+    var vm = this;
+    vm.formInfo = {};
+    vm.formInfo.methods = [];
+    vm.formInfo.body = {};
+    vm.routes = routes.routes;
+    vm.user = {};
```

Comment:
do we need to make a duplicate of user or can we keep the user on the user factory?